src/scripts/main.js

```diff
@@ -1,3 +1,87 @@
 'use strict';
 
-// write code here
+const container = document.querySelector('.container');
+const appendRow = container.querySelector('.append-row');
+const appendCol = container.querySelector('.append-column');
+const removeRow = container.querySelector('.remove-row');
+const removeCol = container.querySelector('.remove-column');
+const table = container.querySelector('table');
+
+let countRow = table.rows.length;
+let countCol = table.rows[0].cells.length;
+
+const appendRowClick = () => {
+  const newRow = table.rows[0].cloneNode(true);
+
+  table.appendChild(newRow);
+};
+const appendColClick = () => {
+  for (let i = 0; i < countRow; i++) {
+    const newCol = table.rows[0].cells[0].cloneNode(true);
+
+    table.rows[i].appendChild(newCol);
+  }
+};
+const removeRowClick = () => {
+  table.deleteRow(0);
+};
+const removeColClick = () => {
+  for (let i = 0; i < countRow; i++) {
+    table.rows[i].deleteCell(0);
```

Comment:
The `deleteCell` method is used correctly here to delete the first cell in each row. Ensure that this logic aligns with the intended functionality, as deleting the first cell might not always be the desired action.